CHANGELOG.md

```diff
@@ -4,15 +4,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 0.2.1 - 2019-10-04
+## [Unreleased]
+
+## [0.3.0] - 2019-10-17
+
+### Added
+- Deploy an ERC20 contract on the Ethereum node and give tokens to both parties. The address of the token contract is written to the env file as `ERC20_CONTRACT_ADDRESS`.
+
+### Changed
+- Migrate parts of the code base to async/await :tada:
```

Comment:
```suggestion
- Migrate parts of the code base to async/await 🎉.
```

Reply:
But I like the text-emoji!
And it renders on GitHub :D

Reply:
It does not render in CLion....